quantum/process_keycode/process_audio.h

```diff
@@ -0,0 +1,11 @@
+#ifndef PROCESS_AUDIO_H
+#define PROCESS_AUDIO_H
+
+bool process_audio(uint16_t keycode, keyrecord_t *record);
+void process_audio_noteon(uint8_t note);
+void process_audio_noteoff(uint8_t note);
+void process_audio_stop_all_notes(void);
```

Comment:
this should probably be named `process_audio_all_notes_off` to align better with names used in `process_music.c`